src/main/ts/component/Editor.ts

```diff
@@ -14,17 +14,16 @@ enum Status {
 const parseJsonResolveGlobals = (value: string): any => {
   try {
     return JSON.parse(value);
-  } catch(e) { /* ignore */ }
+  } catch (_e) { /* ignore */ }
   return Resolve.resolve(value);
 };
 const lookup = (values: Record<string, any>) => (key: string) => Obj.has(values, key) ? values[key] : key;
 
 const parseGlobal = Resolve.resolve;
 const parseString = Fun.identity;
-const parseFalseOrString = lookup({ 'false': false });
-const parseBooleanOrString = lookup({ 'true': true, 'false': false });
-const parseNumberOrString = (value: string) => /^\d+$/.test(value) ? Number.parseInt(value, 10) : value
-
+const parseFalseOrString = lookup({ false: false });
```

Comment:
Hmm, bare keywords make me suspicious. I suppose we don't support IE anyway but this definitely won't work on IE.

Reply:
So I'd prefer if we always quoted the property names if that's what we need to do to be consistent.

Reply:
Testing in IE11 - it works fine from IE9 mode onwards. 

In any case, quote-props: consistent is a pretty weird rule. I might just turn it off here.

Reply:
I've set it to the below (see https://eslint.org/docs/rules/quote-props). I think this is a bit more of a sensible value.

```
    "quote-props": [
      2,
      "as-needed",
      { "keywords": true }
    ]
```